packages/hs-react-ui/src/components/Button/Button.tsx

```diff
@@ -35,7 +33,7 @@ export type ButtonProps = {
   StyledContainer?: string & StyledComponentBase<any, {}>;
   icon?: any;
   isLoading?: any;
-  children?: string | Node;
+  children?: ReactNode;
```

Comment:
I've been just putting `any` here since we could potentially let them pass in an array or ReactNodes as well

Reply:
I believe ReactNode covers everything:
https://github.com/typescript-cheatsheets/react-typescript-cheatsheet/blob/cc2ec67e0b36b7e838df8f2a54b4cd84df6c78cd/README.md#useful-react-prop-type-examples